wrangler-core/src/main/java/co/cask/wrangler/internal/TextDirectives.java

```diff
@@ -696,6 +698,14 @@ public TextDirectives(String directives) {
         }
         break;
 
+        // columns-format-case <current-case> <desired-case>
+        case "columns-format-case" : {
+          String currentCase = getNextToken(tokenizer, command, "current-case", lineno);
+          String desiredCase = getNextToken(tokenizer, command, "desired-case", lineno);
```

Comment:
What happens if I mis-type the desired-case. There is no validation on the supported types. Could we add check for that and in case it doesn't match the things that we want it to be we throw a ```DirectiveParseException```